src/components/originalSampleProcessing/ProcessingSuccess.tsx

```diff
@@ -11,13 +11,20 @@ import { Column } from 'react-table';
 import Heading from '../Heading';
 import { ModalBody } from '../Modal';
 import BlueButton from '../buttons/BlueButton';
+import { reload } from '../../lib/sdk';
+import WhiteButton from '../buttons/WhiteButton';
+import { createSessionStorageForLabwareAwaiting } from '../../types/stan';
 interface ProcessingSuccessProps {
   labware: LabwareFieldsFragment[];
   columns: Column<LabwareFieldsFragment>[];
   successMessage: string;
 }
 
 const ProcessingSuccess: React.FC<ProcessingSuccessProps> = ({ labware, columns, successMessage }) => {
+  const navigate = useNavigate();
+  React.useEffect(() => {
+    createSessionStorageForLabwareAwaiting(labware);
+  }, [labware]);
```

Comment:
ProcessingSuccess is the component called when the BlockProcessing or/and PotProcessing operations succeed,   in means that the store option should be added to this component and the operations related, otherwise will need to  add the createSessionStorageForLabwareAwaiting in both components (BlockProcessing, PotProcessing)

Reply:
Yes, I can see the point. If it only that in case  when it is called from another context which doesn’t require session storage update in future, especially the name of the component feels like it is only just displaying success  message. 
But , may be we can leave it especially as it is only called only in specific context( Block and Pot Processing) . It would be good to  add a comment/documentation  on top the component which warns of this session storage update.